api/descriptions/policies/vulnerability_ci_images_get.md

```diff
@@ -0,0 +1,72 @@
+Retrieves the vulnerability rules to raise alerts or block deployments when images with vulnerabilities are detected.
```

Comment:
@KICKdesigns We scan a lot of different resource types, so I'd like our endpoint descriptions to be clear where this policy is applied.

In this case, this policy targets a customer's CI pipeline. We've got a native Jenkins plugin (for customers whose CI is built on Jenkins). For all other CIs, where we don't have a native plugin (e.g. CircleCI, AWS CodeBuild, etc), we offer a command line tool called twistcli. The policy set in this endpoint targets the scanners in the Jenkins plugin and twistcli.

Just for some perspective: The idea is that we can give developers an early look into the vulnerabilities of the components they've included in the container images they're building. The security team sets the policy and the developer's build tool will fail the build if it doesn't meet the security bar. Essentially, the scanner runs as post-build step and assesses the build artifact (a container image). This whole idea of giving devs an early look into security problems while the software is being built is called "Shift left".

Reply:
@iansk I've updated the doc to include this information. Should it just apply to the PUT or the GET also. As for getting the target resource info (in general for the other policy endpoints), what set of docs can I search to look this information up?